disco/postprocess/hosting_capacity.py

```diff
@@ -167,6 +167,12 @@ def compute_hc_per_metric_class(
         metric_df = metric_df[metric_df.node_types == node_types[0]]
 
     queries = build_queries(metric_df.columns, thresholds, metric_class, on=on)
+
+    if metric_class == "thermal":
+        if (metric_df.transformer_instantaneous_threshold.isna().any() or 
```

Comment:
If I am reading this correctly, it will remove transformer checks from all rows if any row has `transformer_instantaneous_threshold = null`. A row with a transformer violation would not be caught. I think this would work if you built one query per feeder (inside `get_hosting_capacity`). Or you could enhance the query.

Reply:
Good catch, @daniel-thom!
I pushed a new fix. Please, check it out and let me know what you think.